src/pages/home/ReportScreen.js

```diff
@@ -140,13 +137,18 @@ class ReportScreen extends React.Component {
 
         this.flatListRef = React.createRef();
         this.reactionListRef = React.createRef();
+
+        // We need unique ID for drag and drop to work properly with stack navigator.
+        this.dragAndDropId = CONST.REPORT.DROP_NATIVE_ID + ReportUtils.generateReportID();
```

Comment:
Should we just use this reports id? they should be unique too

Reply:
You can have two or more of the same reports on the stack